tests/util_tests/test_publish.py

```diff
@@ -478,6 +479,87 @@ def test_case_difference(self, get_local_storage_roots):
         self.assertEqual("primary", root_name)
         self.assertEqual(expected, path_cache)
 
+    @patch("tank.pipelineconfig.PipelineConfiguration.get_local_storage_roots")
+    def test_path_normalization_win_drive_letter(self, get_local_storage_roots):
+        """
+        Ensures that a variety of different slash syntaxes are valid when splitting
+        a path into a storage + path cache field while using a windows drive letter path.
+        """
+        if sys.platform != "win32":
```

Comment:
How about we introduce a decorator that skips the test on windows instead? `@skip_if_windows` This way we would have a clean way to document tests being skipped and it would be easy to look for them.

Reply:
good idea! done!!!